packages/core/src/balance/Balance.chain.ts

```diff
@@ -86,6 +86,8 @@ export async function listenToBalanceChanges(
 
   return blockchain.api.query.system.account(
     accountAddress,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
Q: Does this not compile?

Reply:
It used to generate lint complaints, but now it does not, so I guess I can take it off.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/434/commits/237a30863f63ce624399146463288a409348658e